Submitted By: Jim Gifford (patches at jg555 dot com)
Date: 2003-09-09
Initial Package Version: 2.8.4
Origin: taviso@gentoo.org Gentoo Bug # 24238
Description: --tabsize option, undocumented in diff but used in sdiff, makes
	     diff dump core

diff -Naur diffutils-2.8.4.orig/src/diff.c diffutils-2.8.4/src/diff.c
--- diffutils-2.8.4.orig/src/diff.c	2002-06-17 05:55:42.000000000 +0000
+++ diffutils-2.8.4/src/diff.c	2003-09-09 20:29:43.000000000 +0000
@@ -234,7 +234,7 @@
   {"starting-file", 1, 0, 'S'},
   {"strip-trailing-cr", 0, 0, STRIP_TRAILING_CR_OPTION},
   {"suppress-common-lines", 0, 0, SUPPRESS_COMMON_LINES_OPTION},
-  {"tabsize", 0, 0, TABSIZE_OPTION},
+  {"tabsize", 1, 0, TABSIZE_OPTION},
   {"text", 0, 0, 'a'},
   {"to-file", 1, 0, TO_FILE_OPTION},
   {"unchanged-group-format", 1, 0, UNCHANGED_GROUP_FORMAT_OPTION},
